charts/neon-storage-controller/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: neon-storage-controller
 description: Neon storage controller
 type: application
-version: 1.2.1
+version: 1.2.3
```

Comment:
This could be `1.3.0` because it's not just a bugfix and we're actually changing services.

Reply:
Just from the point of a end user of the Helm chart, even if they don't change their values a new service will be deployed, or am I mistaken? That sounds like a behavior change warranting bumping the minor version number to me, but it's also no big deal to just bump the patch number. However, it should be `1.2.2`, no? Unless that was skipped for a reason.

Reply:
yes, it was a typo that I used 1.2.3 instead of 1.2.2. anyways I've updated it to 1.3.0, let me know if we can keep this?